app/Http/Controllers/Import/ImportWizardController.php

```
@@ -1,63 +1,104 @@
         $request->validated();
-        Log::debug('File validation passed');
 
         $file = $request->getFile();
-        $originalFilename = $file->getClientOriginalName();
-        $filename = Str::random(40).'.csv';
-        Log::debug('Generated filename', ['original' => $originalFilename, 'generated' => $filename]);
 
-        // Preprocess the CSV file to ensure proper UTF-8 encoding
-        $preprocessedPath = (new CsvProcessor)->preprocessCSV($file, $request->getDelimiter(), $request->getQuoteChar());
-        Log::debug('CSV file preprocessed', ['path' => $preprocessedPath]);
+        if (! $file->isValid()) {
```

PHPStan (2.1.15)

Cannot call method isValid() on Illuminate\Http\UploadedFile|null.


(method.nonObject)